core-library/src/test/java/org/silverpeas/core/contribution/ContributionModificationContextHandlerTest.java

```diff
@@ -124,8 +124,6 @@ void minorModificationWhenHttpParameterIsEqualToTrue() {
         "{\"isMinor\":true}");
     ContributionOperationContextPropertyHandler.parseRequest(request);
     assertMinorModificationDetected();
-    CacheServiceProvider.getThreadCacheService().clearAllCaches();
```

Comment:
Because the additional cache cleaning was added here  to circumvent a bug with `ThreadCache`: `ThreadCache` used a `ThreadLocal` instance attribute instead of a class one (id est static one). It is strongly recommended the `ThreadLocal` vars to be static unless unexpected behavior can occurs when using them.

Reply:
getThreadCacheAccessor is now the unique way to access a cache linked to a thread, and so to a request.